core/arch/arm/plat-imx/imx-common.c

```diff
@@ -13,34 +13,43 @@
 #include <mm/core_memprot.h>
 #include <platform_config.h>
 
-static uint32_t imx_digproc(void)
+static int imx_cpu_type = (-1);
+static int imx_soc_revision = (-1);
```

Comment:
this mandates major < 0x80.
minor: drop parentheses

Reply:
Addressed in a5496ac